pkg/cmd/version.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+)
+
+const version = "0.1.0"
+
+// VersionCommand show the current version of tekton cli.
+func VersionCommand(p cli.Params) *cobra.Command {
+	versionCmd := &cobra.Command{
+		Use:   "version",
+		Short: "Prints the cli version",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Printf("Version: %s\n", version)
```

Comment:
Please consider using `    fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", ClientVersion) `
As there can be a Tekton pipeline version we can show as an improvement

Reply:
Sure